Upgrade to version 1.3.0

danjee opened the pull request:
Upgrade to version 1.3.0 (needed for jquery 1.12.0)
[jquery forum](https://forum.jquery.com/topic/typeerror-view-is-null-line-5645-of-jquery-2-2-0-js)

Commit:
upgrade version to 1.3.0 release

Upgrade to 1.3.0 release

Commit:
Merge pull request #1 from danjee/upgrade_1_3_0

upgrade version to 1.3.0 release

jamesward commented on pom.xml:
```diff
@@ -11,7 +11,7 @@
     <groupId>org.webjars</groupId>
     <artifactId>jquery-migrate</artifactId>
-    <version>1.2.2-SNAPSHOT</version>
+    <version>1.3.1-SNAPSHOT</version>
```
This needs to be `1.3.0-SNAPSHOT`.

Commit:
Update pom.xml

I've put 1.3.1, having thought that this will be next iteration. Sorry for misunderstanding.
